src/core/distributions/phylogenetics/tree/birthdeath/EpisodicBirthDeathProcess.h

```diff
@@ -13,21 +13,28 @@ namespace RevBayesCore {
         
     public:
         EpisodicBirthDeathProcess(const TypedDagNode<double> *ra,
-                                  const TypedDagNode<RbVector<double> > *s, const TypedDagNode<RbVector<double> > *st,
-                                  const TypedDagNode<RbVector<double> > *e, const TypedDagNode<RbVector<double> > *et,
-                                  const TypedDagNode<double> *r, const std::string& ss, const std::vector<Clade> &ic,
-                                  const std::string &cdt, const std::vector<Taxon> &tn);
+                                  const TypedDagNode<RbVector<double> > *s,
+                                  const TypedDagNode<RbVector<double> > *st,
+                                  const TypedDagNode<RbVector<double> > *e,
+                                  const TypedDagNode<RbVector<double> > *et,
+                                  const TypedDagNode<double> *r,
+                                  const TypedDagNode<double> *mp,
+                                  const std::string& ss,
+                                  const std::vector<Clade> &ic,
+                                  const std::string &cdt,
+                                  const std::vector<Taxon> &tn,
+                                  Tree *t);
         
         // public member functions
-        EpisodicBirthDeathProcess*                          clone(void) const;                                                                      //!< Create an independent clone
+        EpisodicBirthDeathProcess*                          clone(void) const;                                                              //!< Create an independent clone
         
         
     protected:
         // Parameter management functions
-        void                                                swapParameterInternal(const DagNode *oldP, const DagNode *newP);                        //!< Swap a parameter
-//        double                                              computeLnProbabilityTimes(void) const;                                                              //!< Compute the log-transformed probability of the current value.
-        double                                              lnProbNumTaxa(size_t n, double start, double end, bool MRCA) const { throw RbException("Cannot compute P(nTaxa)."); }
-        void                                                prepareProbComputation(void);
+        void                                                swapParameterInternal(const DagNode *oldP, const DagNode *newP);                //!< Swap a parameter
+//        double                                              computeLnProbabilityTimes(void) const;                                         //!< Compute the log-transformed probability of the current value.
+//        double                                              lnProbNumTaxa(size_t n, double start, double end, bool MRCA) const { throw RbException("Cannot compute P(nTaxa)."); }
```

Comment:
commented out functions can just be removed

Reply:
Done, see commit 068ea9105f4dfc96cb18b414809dadaa8acdf2b9